hicommonwealth/commonwealth
# Open Graph Meta tags for Thread details page

## Link to Issue
Closes: #6638 

## Description of Changes
- Added Helmet async
- Added some functionality to generate the correct meta tags in thread details page
- Made `getMetaDescription` function in order to parse legacy Quill deltas in thread details page

## Test Plan

Via Frick
- Go to [https://commonwealth-frick.herokuapp.com/](Frick)
- Go to a thread
- Share via any platform and ensure it's correct

If frick/frack is unavailable, then it can also be tested via IP tunneling. I am using [this chrome extention](https://chromewebstore.google.com/detail/localhost-open-graph-chec/gcbnmkhkglonipggglncobhklaegphgn) with https://www.opengraph.xyz/ which would give a preview of the link on facebook, discord, LinkedIn and twitter. For twitter, https://www.bannerbear.com/tools/twitter-card-preview-tool/#image_result is a better tool
